src/compiler/checker.ts

```diff
@@ -8350,10 +8650,7 @@ namespace ts {
         }
 
         function getJsxGlobalElementClassType(): Type {
-            if (!jsxElementClassType) {
-                jsxElementClassType = getExportedTypeFromNamespace(JsxNames.JSX, JsxNames.ElementClass);
-            }
-            return jsxElementClassType;
+            return getExportedTypeFromNamespace(JsxNames.JSX, JsxNames.ElementClass);

```

Comment:
Are the above supposed to be in this change? (See removal of var on line 167 also).